include/swift/AST/Module.h

```diff
@@ -197,6 +197,7 @@ struct ImplicitImportInfo {
 
 class OverlayFile;
 
+
```

Comment:
Could you clean up the stray whitespace?

Reply:
resolved in af9a745